### PR TITLE
Update dependencies

### LINKED ISSUE #2: Update chloride
This currently uses chloride 2.2.14.  It would be nice if it could use a current version of chloride so we don't have to have multiple copies of chloride and sodium-native included in ssb-browser-demo.

### PR DESCRIPTION
**Pull request by KyleMaas**
Fixes #2.  Updated other dependencies as well.  Passes tests.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "ssb-keys-mnemonic",
   "description": "Module that converts from/to SSB keys and BIP39 mnemonic codes",
-  "version": "1.0.0",
+  "version": "1.0.1",
```
Comment (staltz) [inline package.json]: I usually do a separate commit for the version. Could you revert this one?

Comment (KyleMaas) [reply package.json]: Done.

**Comment by staltz** [comment]
Thanks
